src/Utilities/Compiler/FxCopWellKnownDiagnosticTags.cs

```diff
@@ -16,12 +13,5 @@ internal static class FxCopWellKnownDiagnosticTags
 
         public static readonly string[] PortedFxCopDataflowRule = new string[] { PortedFromFxCop, WellKnownDiagnosticTagsExtensions.Dataflow, WellKnownDiagnosticTags.Telemetry };
         public static readonly string[] PortedFxCopDataflowRuleEnabledInAggressiveMode = new string[] { PortedFromFxCop, WellKnownDiagnosticTagsExtensions.Dataflow, WellKnownDiagnosticTags.Telemetry, WellKnownDiagnosticTagsExtensions.EnabledRuleInAggressiveMode };
-
-        public static bool IsPortedFxCopRule(DiagnosticDescriptor diagnosticDescriptor)
-        {
-            var result = diagnosticDescriptor.CustomTags.Any(t => t == PortedFromFxCop);
-            Debug.Assert(!result || diagnosticDescriptor.Id.StartsWith("CA", StringComparison.OrdinalIgnoreCase));
-            return result;
-        }
```

Comment:
This is not currently used.

It may have been used in the deleted `FxCopAnalyzers.sln`.